src/odemis/acq/stitching/_registrar.py

```diff
@@ -107,131 +114,138 @@ def addTile(self,tile,dependent_tiles = None):
-            ver_diff = md_pos[1]-md_pos_prev[1]
-            hor_diff = md_pos[0]-md_pos_prev[0]
+
+            # Insert new tile either to the right or to the bottom of the
+            # closest tile.
+            ver_diff = md_pos[1] - md_pos_prev[1]
+            hor_diff = md_pos[0] - md_pos_prev[0]
             if abs(ver_diff) > abs(hor_diff):
                 if ver_diff < 0:
                     self.posY = pos_prev[0] + 1
                     self.posX = pos_prev[1]
                     if len(self.shifts) <= self.posY:
-                        self._updateGrid("y") # extend grid in y direction
+                        self._updateGrid("y")  # extend grid in y direction
                 else:
                     self.posY = pos_prev[0] - 1
                     self.posX = pos_prev[1]
             else:
                 if hor_diff > 0:
                     self.posY = pos_prev[0]
-                    self.posX = pos_prev[1] + 1    
+                    self.posX = pos_prev[1] + 1
                     if len(self.shifts[0]) <= self.posX:
-                        self._updateGrid("x") # extend grid in x direction
+                        self._updateGrid("x")  # extend grid in x direction
                 else:
                     self.posY = pos_prev[0]
-                    self.posX = pos_prev[1] - 1 
-            
+                    self.posX = pos_prev[1] - 1
+
             self.pos_prev_x = pos_prev[1]
             # Calculate overlap
             if abs(ver_diff) > abs(hor_diff):
                 size_meter = self.size[0] * self.px_size[1]
                 diff = abs(tile.metadata[model.MD_POS][1] - md_pos_prev[1])
                 self.ovrlp = abs(size_meter - diff) / size_meter
                 # self.size is given as YX, MD_POS as XY
-                self.osize = self.size[0] * self.ovrlp  # overlap size in pixels  
+                self.osize = self.size[0] * \
+                    self.ovrlp  # overlap size in pixels
                 self.tsize = int(self.size[0] - self.osize)
             else:
                 size_meter = self.size[1] * self.px_size[0]
                 diff = abs(tile.metadata[model.MD_POS][0] - md_pos_prev[0])
-                self.ovrlp = abs(size_meter-diff) / size_meter
-               
-                self.osize = self.size[1] * self.ovrlp  # overlap size in pixels  
+                self.ovrlp = abs(size_meter - diff) / size_meter
+
+                self.osize = self.size[1] * \
+                    self.ovrlp  # overlap size in pixels
                 self.tsize = int(self.size[1] - self.osize)
-                
+
         if dependent_tiles != None:
-            for i in range(len(dependent_tiles)):                 
-                self.shift_tile_dep_tiles[i].append(numpy.subtract(dependent_tiles[i].metadata[model.MD_POS], \
-                                                                tile.metadata[model.MD_POS]))
+            for i in range(len(dependent_tiles)):
```

Comment:
This is a too long line. I would first put the subtraction on a line, and then the append. You could also improve the readeability by changing the `for ... range()` into a `for a,b ... zip()`.  So I'd suggest to write it this way:
```
for shift_dt, dt in zip(self.shift_tile_dep_tiles, dependent_tiles):
         sdt = numpy.subtract(dt.metadata[model.MD_POS], tile.metadata[model.MD_POS])
         shift_dt.append(sdt)
```

Also, as a digression, you need to be careful with using numpy in such case, because it returns a numpy array, which behaves often enough like another iterable, but not always. In particular, things like this don't behave as you'd expect: `a == b`, if a or b are a numpy array. (It doesn't return a boolean, but an array of booleans, unless they are different length.) In general, I'd just write such subtraction with `tuple(a - b for a, b in zip(va, vb))`. Anyway, I guess in this case, it's safe to do so, especially as it seems shift_tile_dep_tiles is only used internally.